src/main/resources/templates/fragments/include.html

```diff
@@ -27,12 +24,18 @@ <h1 class="font-weight-bold text-dark pt-2 mb-0 d-inline-block">Include ({{group
                 <tr>
                     <th scope="col" class="ml-1 align-top" aria-label="Check all checkboxes">
                         <input class="position-relative ml-0" type="checkbox"
-                               aria-label="Check all checkboxes"
-                               ng-model="allSelected"
+                               aria-label="Check all in PAGE"
```

Comment:
Let's change the aria-label to "Select all checkboxes in the current page" to make it more understandable for screen readers.